app/components/post_body/post_body.js

```diff
@@ -202,12 +202,12 @@ export default class PostBody extends PureComponent {
         }
 
         const postId = postProps.add_channel_member.post_id;
-        const noGroupsUsernames = postProps.add_channel_member?.not_in_groups_usernames;
-        let userIds = postProps.add_channel_member?.not_in_channel_user_ids;
-        let usernames = postProps.add_channel_member?.not_in_channel_usernames;
+        const noGroupsUsernames = postProps.add_channel_member?.not_in_groups_usernames; // eslint-disable-line camelcase
```

Comment:
any reason why we don't want camelcase ?

Reply:
Those post props come from the server, so they're allowed to be in snake_case. There's a bug with this ESLint rule where it breaks when using the optional chaining operator though, so this comment is required when we mix the two. Hopefully this won't be necessary in the future once we update some more of our dependencies.